src/Provider/Provider.Web/ViewModels/Reports/ProviderApplicationsReport/ProviderApplicationsReportCreateEditModelValidator.cs

```diff
@@ -31,11 +30,13 @@ public ProviderApplicationsReportCreateEditModelValidator(ITimeProvider timeProv
                 RuleFor(x => x.ToDate)
                     .Cascade(CascadeMode.StopOnFirstFailure)
                     .Must(date => date.AsDateTimeUk() < timeProvider.NextDay)
-                    .WithMessage("Date to cannot be in the future");
+                    .WithMessage("Date to cannot be in the future")
```

Comment:
This reads weird, Date To I am assuming can only be any date up to today/Now of when this report is being created.